scripts/test-run-mysql

```diff
@@ -23,11 +23,11 @@ start-test() {
 export -f start-test
 
 VERSION_LIST="\
+    mysql 8.4
     mysql 8.0
-    mysql 5.7
+    mariadb 11.4
```

Comment:
looks like the connection test doesn't work for 11.4, I suspect something's changed in the CLI or password env var stuff.